src/FtlPlayer.ts

```diff
@@ -1,4 +1,4 @@
-import { Janus, PluginHandle, Message, JSEP } from "janus-gateway";
+import { Janus, PluginHandle, Message, JSEP } from "janus-gateway-mirror";
```

Comment:
Ideally these would stay the same, but I could not find any sane way of doing it with light googling.